setup.cfg

```diff
@@ -32,6 +32,10 @@ install_requires =
     invenio-administration>=2.0.0,<3.0.0
     lxml>=4.5.0
     PyYAML>=5.4.1
+    awscli>=1.33.23
```

Comment:
are all these needed?

Reply:
Probably awscli could be removed but the rest were needed, however due to dependency issues between boto3, botocore and commonmet-py, I have replaced them with s3fs which seems to work as fine.